render-test/runner.cpp

```diff
@@ -76,7 +76,7 @@ std::string simpleDiff(const Value& result, const Value& expected) {
             break;
         }
 
-        if (!deepEqual(resultTokens[i], expectedTokens[i])) {
+        if (resultTokens[i] != expectedTokens[i]) {
```

Comment:
These are `std::string`s. I was getting an error that the call to `deepEqual` was ambigious (probably because `std::optional` can be implicitly constructed).